### PR TITLE
docs: tip to intercept all queries

### PR DESCRIPTION
**Pull request by QPierre1**
Share a workaround to intercept all requests

re #6099

Signed-off-by: quentin <31855566+QPierre1@users.noreply.github.com>

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by achrinza** [comment]
Thanks for the pull request! Please refrain from opening duplicate PRa and consider using `git rebase` and `git push --force-with-lease` to fix the git history.

File: docs/site/Interceptor.md [hunk 1]
```diff
@@ -360,6 +360,8 @@ The registration can be further simplified as:
 app.interceptor(MetricsInterceptorProvider, {global: true, group: 'metrics'});
 ```
 
+{% include tip.html content="If you need to intercept all requests, including these destinated to a LB3 mounted app, you can use [a Loopback middleware](https://loopback.io/doc/en/lb4/Middleware.html#loopback-style-middleware)" %}
```
Comment (achrinza) [inline docs/site/Interceptor.md:363]: Apologies for missing this PR, may I suggest that we reference the Markdown file directly? The docs generator will automatically convert it into the relevant URLs:
```suggestion
{% include tip.html content="If you need to intercept all requests, including these destinated to a LB3 mounted app, you can use a [LoopBack Middleware](./Middleware.md#loopback-style-middleware)" %}
```

**Review by achrinza — CHANGES_REQUESTED** [review]

**Comment by QPierre1** [comment]
Yes

**Comment by achrinza** [comment]
Thanks for the Pull Request! It's been merged to master.

**Pull request merged**